docs/config/README.md

```diff
@@ -91,7 +111,7 @@ The example above generates `John-SampleApp` as the prefix for stacks, roles, pi
 
 ### Accounts Section
 The accounts section of the config file is a collection of account objects that can be used to deploy apps and run cicd. The accounts also represent environments.
-We define three accounts in the sample config file above: `staging`, `production`, `cicd`, each with different account numbers.
+We define four accounts in the sample config file above: `staging`, `production`, `cicd`, `domain` each with different account numbers.
```

Comment:
This part confuses me a bit.  AFAIK the configs are meant to be 1:1 with environments.  The `domain` as an added environment seems wonky, or I at least don't understand the "why" without reading more here.

Reply:
I think the main idea of Mira is to have 5 accounts: a `domain` for managing domain settings, a `cicd` for the pipeline and `dev`, `staging` and `prod` for the respective environments. The domain management can live only in one of the other accounts (ie the `prod` account`) it is a better practice to have it as a separate account.

Reply:
So why not only deploy it to prod I guess?  Would we ever want to deploy subdomains i.e. `dev.nearform.com` in a different env?  How does the `domain` env interface with that?

Reply:
The Route53 manager needs to be deployed into the account that has the domain name registered. This can be a special `domain` account or it can be the `prod` as well. It doesn't matter. Then the Certificate manager must be deployed into each (dev, staging, prod, and any other environment) account because the certificate must live in the account that uses the certificate.